kythe/java/com/google/devtools/kythe/analyzers/java/KytheDocTreeScanner.java

```diff
@@ -56,11 +57,23 @@ public KytheDocTreeScanner(KytheTreeScanner treeScanner, Context context) {
     this.trees = JavacTrees.instance(context);
   }
 
-  public boolean visitDocComment(TreePath treePath, VName node, EntrySet absNode) {
+  @AutoValue
+  abstract static class DocCommentVisitResult {
+    abstract boolean documented();
+
+    abstract Optional<String> deprecated();
```

Comment:
deprecation() ? Seems a bit odd to have a non-boolean named "deprecated".

Reply:
Sure, done